pkg/btree/LICENSE

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
We need keep LICENSE.

Reply:
The LICENSE is contained in the go file.